content/en/tracing/guide/tutorial-enable-python-containers.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Tutorial - Enabling Tracing for a Python Application and Datadog Agent in Containers
+kind: guide
+further_reading:
+- link: /tracing/trace_collection/library_config/python/
+  tags: Documentation
+  text: Additional tracing library configuration options
+- link: /tracing/trace_collection/dd_libraries/python/
+  tags: Documentation
+  text: Detailed tracing library setup instructions
+- link: /tracing/trace_collection/compatibility/python/
+  tags: Documentation
+  text: Supported Python frameworks for automatic instrumentation
+- link: /tracing/trace_collection/custom_instrumentation/python/
+  tags: Documentation
+  text: Manually configuring traces and spans
+- link: https://github.com/DataDog/dd-trace-py
+  tags: GitHub
+  text: Tracing library open source code repository
+---
+
+## Overview
+
+This tutorial walks you through step by step enabling tracing on a sample Python application that is installed in a container. In this scenario, the Datadog Agent is also installed in a container. 
+
+For other scenarios, including application and Agent on a host, the application in a container and Agent on a host, and applications written in other languages, see the other [Enabling Tracing tutorials][1].
+
+For general comprehensive tracing setup documentation for Python, see [Tracing Python Applications][2].
```

Comment:
```suggestion
See [Tracing Python Applications][2] for general comprehensive tracing setup documentation for Python.
```

Reply:
oops, I forgot that you told me not to repeat feedback. I'll just say "same as above" now